packages/ChangesReloaded-Core.package/CRBrowser.class/instance/changeListItems..st

```diff
@@ -0,0 +1,4 @@
+accessing
+changeListItems: aSortedCollection
```

Comment:
Does this still have to be a `SortedCollection`, or just an `OrderedCollection`?

Reply:
True, it does not have to be one